backend/src/api/album/get_filters.rs

```diff
@@ -0,0 +1,238 @@
+use anyhow::Context;
+use axum::{extract::Query, Extension, Json};
+use itertools::Itertools;
+use serde::{de::DeserializeOwned, Serialize};
+use serde_rusqlite::from_row;
+
+use std::sync::Arc;
+
+use crate::api::{auth::Authorize, error::Error};
+use crate::{AppState, DbInteractable, SqliteDatabase};
+
+use super::{AlbumFilters, Timeframe};
+
+#[derive(Debug, Default, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub(super) struct AvailableAlbumFilters {
+    authors: Vec<String>,
+    timeframes: Vec<Timeframe>,
+    has_drafts: bool,
+}
+
+// This function returns available filters.
+// If filters are sent in it returns the available filters that can be used alternatively.
+// As an example, the returned authors are all authors that can be found
+// with all other filters applied. If the author is filtered on that is ignored for the
+// returned author since otherwise it would only return one value.
+// The same logic applies to all other filters.
```

Comment:
Fix line breaks and use and empty line for new paragraphs

Reply:
```suggestion
// This function returns available filters.
//
// If filters are sent in it returns the available filters that can be used to further filter the
// result.
//
// For example, the returned authors are all authors that can be found with all other filters
// applied. If the author is filtered on it is ignored for the returned author since otherwise it
// would only return one value. The same logic applies to all other filters.
```